### PR TITLE
added about us

### PR DESCRIPTION
**Pull request by aniagaska**
I fixed the spelling on Estefania's name, added the contact us and about us sections with some styling

I need some help making the meet our team section more cohesive (meet our team is centered and the other headings aren't), and how to make the styling more consistent with the design starter

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -140,9 +143,30 @@ <h2 class="project__title">Astoria Community Bulletin Board</h2>
           </div>
         </div>
 
+    </main><!-- end site-content -->
+
+    <section class="aboutus-section">
+      
+      <div class= "aboutus">
```
Comment (bronzehedwick) [inline index.html]:
```suggestion
      <div class="aboutus">
```

File: index.html [hunk 2]
```diff
@@ -140,9 +143,30 @@ <h2 class="project__title">Astoria Community Bulletin Board</h2>
...
+        <h1>About Us</h1>
+        <p>We are a group of volunteer designers and software engineers from Astoria, Queens. Our mission is to support the work of volunteer groups in our neighborhood with digital tools and create interfaces for civic engagement.</p>
+        <p>Astoria Digital started at the beginning of the covid 19 pandemic in Astoria, Queens. Peter Valdez organized the Astoria Tech meetup that connected software engineers in the neighborhood. When he saw a flier in the neighborhood for Astoria Mutual Aid Network, he reached out to the founder and helped them create a dispatch bot that connected the airtable base with the slack to connect requests from neighbors with volunteers.</p>
+        <p>Since then, we have expanded our projects by collaborating withcreating a guide to completing a FOIL request in New York state. We have continued collaborating with grassroots groups in Astoria by creating the Community Bulletin Board. Our goal is to share the tools we make with other neighborhoods.</p>
       </div>
 
-    </main><!-- end site-content -->
+    </section>
+
+    <section class= "contactus-section" >
```
Comment (bronzehedwick) [inline index.html]:
```suggestion
    <section class="contactus-section">
```

File: styles.css [hunk 3]
```diff
@@ -598,12 +598,38 @@ svg {
...
+  }
+
+  /* }}} */
+
+
+/* Contact Us {{{ */
+
+.contactus-section {
+  padding: 2em;
+  }
```
Comment (bronzehedwick) [inline styles.css]:
```suggestion
}
```

File: index.html [hunk 4]
```diff
@@ -40,9 +40,12 @@ <h1 class="site-header__title">
         <a href="/" class="site-header__text">astoria.digital</a><span class="site-header__cursor">_</span>
       </h1>
       <p class="site-header__slogan">Digital volunteers for the community</p>
-      <a href="mailto:team@astoria.digital?subject=feedback" class="site-header__contact">
+      
+      <!-- <a href="mailto:team@astoria.digital?subject=feedback" class="site-header__contact">
```
Comment (bronzehedwick) [inline index.html:44]: Is this link no longer valid? If it's not should it be completely removed instead of commented out?

File: styles.css [hunk 5]
```diff
@@ -624,12 +650,16 @@ svg {
...
+
+
 /* Footer {{{ */
 
 .site-footer {
   padding: 4rem;
   background-color: var(--highlight-yellow);
   text-align: center;
+  border-top: 3px solid;
+
```
Comment (bronzehedwick) [inline styles.css]:
```suggestion
```

**Review by bronzehedwick — CHANGES_REQUESTED** [review]
Had a question and a few formatting cleanup suggestions. One final question here: should the things you need styling help with be done before this PR is merged, or is this okay as is?

Regardless, let's chat about what you want to do style-wise that you need help with on a call, since it's probably going to be a lot easier to understand that way instead of async github comments.

File: index.html (hunk 4 above)
Comment (aniagaska) [reply index.html:44]: It's a styling idea, instead of having that link in the header I was thinking of including it later in the page

**Review by bronzehedwick — APPROVED** [review]

**Pull request merged**